master/static/migrations/20230920102522_add_login_time.tx.up.sql

```diff
@@ -0,0 +1,13 @@
+ALTER TABLE public.users ALTER COLUMN modified_at TYPE timestamptz;
+ALTER TABLE public.users ADD COLUMN last_login timestamptz NULL;
+
+-- backfill with sessions
+UPDATE public.users u
```

Comment:
seems like a noble attempt to backfill, but is it really necessary? probably fine, but the system still needs to work with null for new users that haven't logged on.

Reply:
I think it's necessary -- while you're right and the code already has to deal with null data, ensuring the data is backfilled now will make exposing the data to end users in the future easier because we won't have to explain when the data was started to be collected.